website/docs/r/satellite_cluster.html.markdown

```diff
@@ -50,6 +50,7 @@ Review the argument references that you can specify for your resource.
 - `location` - (Required, String) The name or ID of the Satellite location.
 - `kube_version` - (Optional, String) The Red Hart OpenShift Container Platform version.
 - `operating_system` - (Optional, String) Operating system of the default worker pool. Options are REDHAT_7_64, REDHAT_8_64, or RHCOS.
+- `entitlement` - (Optional, String) Entitlement reduces additional OCP Licence cost in OpenShift clusters. Use Cloud Pak with OCP Licence entitlement to create the OpenShift cluster. **Note** <ul><li> It is set only when the first time creation of the cluster, further modifications are not impacted. </li></ul> <ul><li> Set this argument to `cloud_pak` only if you use the cluster with a Cloud Pak that has an OpenShift entitlement.</li></ul>.
```

Comment:
This was copied from here, as this option has the exact same behavior as with vpc clusters.
https://github.com/IBM-Cloud/terraform-provider-ibm/blob/ee5af15d48d8e392e09bb4a983039b746379d4bf/website/docs/r/container_vpc_cluster.html.markdown?plain=1#L176

Also, the docs for `satellite_cluster_workerpool` already included the `entitlement` option, even though the code did not support it.